Documentation/UserInterfaceRecipes.md

```diff
@@ -0,0 +1,128 @@
+# User Interface Recipes
+
+### ViewModel / Presenter / ViewData Objects
+
+Regardless of what you call them, or how much responsibility they should (or should not) have in your project, generally these objects contain a 1:1 mapping of properties to the controls on your screen. The type of properties you use (`MutableProperty`, `ConstantProperty`, or `AnyProperty`) depends on your specific needs.
+
+For example, consider a UI that displays both a slider and a text field that represent a magnitude. The text field specifies its values in dBFS (decibels, full scale). A common approach is to expose these values in your ViewModel using only a single `MutableProperty`:
+
+``` swift
+final class VolumeViewModel {
```

Comment:
Yep, I dislike the term as well. 

I've switched to `ViewData` as the default naming for much of my own code and have even kicked around the use of `Presenter` in some spots. I only used `ViewModel` due to its recognition from other frameworks and the "MVVM Craze" though I suppose that will die down over time.

(The joys of a one-man coding team—schizophrenic/inconsistent naming conventions!)

I tend to agree with the overall sentiment from [this article](http://khanlou.com/2015/12/mvvm-is-not-very-good/) so I'd be glad to help spread a better approach to figuring this all out.

Any war stories you and others can share in terms of usage/architecture will be helpful.